docs/pages/getting-started/migrating-to-v5.mdx

```diff
@@ -74,8 +74,7 @@ Some things to note about the new configuration:
 The old configuration file, contained in the API Route (`pages/api/auth/[...nextauth].ts` / `app/api/auth/[...nextauth]/route.ts`), now becomes much shorter. **These exports are designed to be used in an App Router API Route**, but the rest of your app can stay in the Pages Router if you are gradually migrating!
 
 ```ts filename="app/api/auth/[...nextauth]/route.ts"
-import { handlers } from "@/auth"
-export const { GET, POST } = handlers
+export { GET, POST } from '@/auth';
```

Comment:
Lets revert this `handlers: { GET, POST }` change (below as well)

Reply:
sgtm